lightning/src/ln/wire.rs

```diff
@@ -16,6 +16,13 @@
 use ln::msgs;
 use util::ser::{Readable, Writeable, Writer};
 
+/// Maximum Lightning message data length according to
+/// [BOLT-8](https://github.com/lightningnetwork/lightning-rfc/blob/v1.0/08-transport.md#lightning-message-specification):
+/// "The maximum size of any Lightning message MUST NOT exceed 65535 bytes.
+/// A maximum size of 65535 simplifies testing, makes memory management easier,
+/// and helps mitigate memory-exhaustion attacks."
+pub const LN_MAX_MSG_LEN: usize = 65535;
```

Comment:
Could we simply use `std::u16::MAX` given the intention given in BOLT 1?

Reply:
I agree, it makes more sense to define the constant here. `65535` is more convenient to the reader though, so perhaps just `std::u16::MAX; // 65535` for optimal legibility

Reply:
Done